roles/ansible-test/tasks/init_collection.yaml

```diff
@@ -4,8 +4,36 @@
     _test_location: "{{ ansible_test_collection_dir }}/{{ ansible_test_collection_namespace }}/{{ ansible_test_collection_name }}"
     _test_cfg_location: "{{ ansible_test_collection_dir }}/{{ ansible_test_collection_namespace }}/{{ ansible_test_collection_name }}/tests/integration/{{ ansible_test_test_command }}.cfg"
 
+- name: Setup minimum test requirements
+  set_fact:
+    _test_requirements: '-r {{ _test_location }}/requirements.txt -r {{ _test_location }}/test-requirements.txt'
```

Comment:
Can we set a new `ansible_test_test_requirements_parameters` in ansible-test/defaults/main.yaml  with`'-r {{ _test_location }}/requirements.txt -r {{ _test_location }}/test-requirements.txt'` as the default value. This way we can overload the value for each project and we avoid this block? And this would remove the need of this block.